woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1907,6 +1923,33 @@ def test_accessor_drop_errors(sample_df):
 
 
 def test_accessor_rename(sample_df):
+    def assert_underlying_data_same(new_df, original_df):
```

Comment:
Same comment as above - this test is big and complicated. Can we test the `inplace` behavior separately?

Reply:
done!